### DISCUSSION TITLE
Προβλήμα με τον grader

### DISCUSSION THREAD

**Discussion by panospask** [pdp-archive/pdp-archive.github.io#256] [General]
Καλησπέρα.
Νομίζω πως ο grader έχει κάποιο πρόβλημα.
Οι υποβολές: 
https://pdp-archive.com/pdp-judge/submission-status.php?id=5167 για το orchestras και
https://pdp-archive.com/pdp-judge/submission-status.php?id=5164 για το secretpaths
γίνονται fail (χωρίς μαλιστα να προσδιοριζεται το είδος λάθους) ενώ ακριβώς ο ίδιος κώδικας είχε παλαιότερα περάσει στον judge. Μάλιστα η υποβολή για το secretpaths είχε γίνει accepted χθες το απόγευμα.

Fail γίνεται επίσης όποιο άλλο πρόβλημα προσπαθήσω να υποβάλλω (ακόμα και αν είχε περάσει παλαιότερα) εκτός του evencycle.

**Comment by Dim131** [comment]
Έχεις δίκιο, συγγνώμη για αυτό. Τώρα δουλεύει εντάξει για εσένα ;

**Reply by panospask** [reply]
Ναι δουλεύει. Ευχαριστώ.
